### DISCUSSION TITLE
Guidance Needed on Continuing Fine-Tuning Process After Pausing

### DISCUSSION THREAD

**Discussion by binhphamthanh** [SWivid/F5-TTS#278] [Q&A] [answered]
Thank you, SWivid team, for your excellent research contribution.

I am currently fine-tuning a language model in Vietnamese. Before fully committing to the fine-tuning process, I decided to pause after completing Epoch 1 to test if I could resume fine-tuning. Unfortunately, I was unable to continue from where I left off, and I couldn't find any relevant information to assist with this issue.

Could you please provide guidance on this matter? Thank you very much!

**Comment by SWivid** [comment]
Hi @binhphamthanh , thanks for recognition of our works,

Thought probably you were using a small set which has few updates in 1 epoch
check you set a proper number with
https://github.com/SWivid/F5-TTS/blob/e963929b8e6961a37f8321b788ca1969ad9ac4e0/src/f5_tts/train/train.py#L34-L35
similar to finetune scripts

If want wandb logging resume, just filling the id of last run to 
https://github.com/SWivid/F5-TTS/blob/e963929b8e6961a37f8321b788ca1969ad9ac4e0/src/f5_tts/train/train.py#L39

F5-TTS Team

**Reply by binhphamthanh** [reply]
Thank you @SWivid  for your quick response.

To clarify, I am fine-tuning through the Gradio UI, using the Auto Settings option in the Train Data tab with a dataset of 29,185 WAV files, saving checkpoints after every 2,918 steps. After completing Epoch 1 and saving the model at step 5,836, I decided to pause the process. I then attempted to resume fine-tuning from this saved point to see if I could continue training from the next step. Unfortunately, the process restarted from the beginning.

Therefore, I’m seeking guidance on how to continue training with the same dataset configuration and settings, starting from step 5,837.

**Reply by SWivid** [reply]
Hi @binhphamthanh , I'm not sure about that. Maybe #57 will help.

**Reply by lpscr** [reply, marked as answer]
@binhphamthanh rename your last `check point.pt` to `last_model.pt` to reconized start train again and must continue for your last check point
